README.md

```diff
@@ -59,4 +59,4 @@ ObjectMapper mapper = JsonMapper.builder()
     .build();
 ```
 
-after which datatype read/write support is available for all normal Jackson operations,
+after which datatype rewad/write support is available for all normal Jackson operations,
```

Comment:
Maybe not this change. :)

Reply:
Good catch. Sorry I've missed it reviewing the diff :/